superset/db_engine_specs/bigquery.py

```diff
@@ -20,16 +20,23 @@
 from typing import Any, Dict, List, Optional, Tuple, TYPE_CHECKING
 
 import pandas as pd
+from flask_babel import gettext as __
 from sqlalchemy import literal_column
 from sqlalchemy.sql.expression import ColumnClause
 
 from superset.db_engine_specs.base import BaseEngineSpec
+from superset.errors import SupersetErrorType
 from superset.utils import core as utils
 
 if TYPE_CHECKING:
     from superset.models.core import Database  # pragma: no cover
 
 
+INVALID_USER_PERMISSIONS = re.compile(
```

Comment:
```suggestion
INVALID_USER_PERMISSIONS _REGEX= re.compile(
```

Reply:
though thinking about it, this should maybe be:
CONNECTION_DATABASE_PERMISSIONS_REGEX in order to mirror the error?